src/inferring-outlive-requirements.md

```diff
@@ -0,0 +1,113 @@
+# Infer Outlives Requirements
+
+This feature allows some outlives requirements to be
```

Comment:
This is written as if it was a feature flag but it's part of stable Rust now and shouldn't be considered as an independent extension.

Reply:
Thanks. Will update language.